spaceflights-pyspark/{{ cookiecutter.repo_name }}/README.md

```diff
@@ -0,0 +1,34 @@
+# {{ cookiecutter.project_name }}
+
+## Overview
+
+This is your new Kedro project, which was generated using `Kedro {{ cookiecutter.kedro_version }}`.
+
+Take a look at the [Kedro documentation](https://docs.kedro.org) to get started.
+
+## Rules and guidelines
+
+In order to get the best out of the template:
+
+* Don't remove any lines from the `.gitignore` file we provide
+* Make sure your results can be reproduced by following a [data engineering convention](https://docs.kedro.org/en/stable/faq/faq.html#what-is-data-engineering-convention)
+* Don't commit data to your repository
+* Don't commit any credentials or your local configuration to your repository. Keep all your credentials and local configuration in `conf/local/`
+
+## How to install dependencies
+
+Declare any dependencies in `src/requirements.txt` for `pip` installation and `src/environment.yml` for `conda` installation.
+
```

Comment:
Correct, I think we should just cover `pip` here since we don't go into details about conda now (removed from https://docs.kedro.org/en/stable/kedro_project_setup/dependencies.html#install-project-specific-dependencies)